api/java/manipulating-databases/db_list.md

```diff
@@ -12,7 +12,7 @@ related_commands:
 # Command syntax #
 
 {% apibody %}
-r.dbList() &rarr; array
+r.dbList() &rarr; DbList
```

Comment:
The return type should be `array`.  The types in this section refer to ReQL types, not Java types.

Reply:
Okay, I will revert that.